client/packages/cli/src/commands/xcmTransfer/index.ts

```diff
@@ -36,21 +38,76 @@ export const handleXcmTransferCommand = async (
 
     try {
         const targetApi = await ApiPromise.create({
-            provider: new WsProvider("ws://127.0.0.1:9933"), // Rococo Validator on Zombienet
+            provider: new WsProvider(args.endpoint), // Rococo Validator on Zombienet
         })
-        const xcmTransactionParams = generateXcmTransferParameters (
-            targetApi,
-            args.dest,
-            args.recipient,
-            "ROC",
-            "para"
-        )
-        console.log("\n")
-        console.table(JSON.stringify(xcmTransactionParams))
+        const xcmBeneficiaryParam = createBeneficiary(targetApi, args.recipient)
+        const xcmAssetFeeItem = targetApi.registry.createType("u32", 0)
+        console.log("Sending XCM Transfer:\n")
+        const keyring = new Keyring({ type: "sr25519" })
+        const signer = keyring.addFromUri("//Alice")
```

Comment:
keyring should be configurable, preferably from env variable. we already use CIRCUIT_SIGNER_KEY for other commands

Reply:
Yup adding that shortly!

Reply:
So added a signer value parameter that can use a custom signer value at [4e9e997](https://github.com/t3rn/t3rn/pull/1350/commits/4e9e997903a5c2b3ce1eb74053778ba0cdd25dbb) . Currently fixing the part that will allow it to use "//Circuit" as a parameter to sign XCM  transactions with CIRCUIT_SIGNER_KEY

Reply:
all errors fixed in [dba8192](https://github.com/t3rn/t3rn/pull/1350/commits/dba8192a1292d9bc3a5b2778ac3b77bf4fabe30d)